tests/integration.rs

```diff
@@ -221,10 +256,7 @@ fn empty_project_output() -> TestResult {
             \),
         \]
         \[DEBUG\] cleaning: ".+debug" with remove_not_built_with_in_a_profile
-        \[DEBUG\] Successfully removed: ".+libsample_project-.+\.rlib"
-        \[DEBUG\] Successfully removed: ".+libsample_project-.+\.rmeta"
-        \[DEBUG\] Successfully removed: ".+sample_project-.+\.d"
-        \[DEBUG\] Successfully removed: ".+.fingerprint.+sample-project-.+"
+        (\[DEBUG\] Successfully removed: (".+libsample_project-.+\.rlib"|".+libsample_project-.+\.rmeta"|".+sample_project-.+\.d"|".+.fingerprint.+sample-project-.+")\n{0,1}){4}
```

Comment:
I agree that regex is kinda bad for this use-case, but I didn't know what to use instead, do you have any suggestions?

I thought I had fixed the order problem now several times, but it struck again.

Maybe this should just expect 4 lines?
```txt
        \[DEBUG\] Successfully removed: ".+sample.+"
        \[DEBUG\] Successfully removed: ".+sample.+"
        \[DEBUG\] Successfully removed: ".+sample.+"
        \[DEBUG\] Successfully removed: ".+sample.+"
```
or
```
(        \[DEBUG\] Successfully removed: ".+sample.+"){4}
```
What do you think? The line you added is a bit weird to read ("weird to read" is almost ironic in a RegEx context).

Reply:
Here's something terrible nobody asked:
```regex
(\[DEBUG\] Successfully removed: .+((lib)?sample).+(rlib|rmeta|d).+){4}
```

Reply:
@marcospb19 Let's just take the first suggestion? =D